source/openpower/keys.rst

```diff
@@ -0,0 +1,106 @@
+.. _keys:
+
+SSH Key Behavior on OpenPOWER
+=============================
+
+These are findings of how SSH Keys behave in various conditions.
```

Comment:
"...various conditions on OSL's POWERDev OpenStack cluster and how to manage keys"